src/components/centered/centered.scss

```diff
@@ -0,0 +1,31 @@
+.page {
+  align-items: center;
+  background-color: $wolox-blue;
+  background-image: url('./../assets/wolox_bg.png');
```

Comment:
maybe this assets has been deleted?

Reply:
no, the asset still exists as a background.